src/run_server.sh

```diff
@@ -97,12 +116,22 @@ function update_folder_permissions() {
     printf "\n### Folder Permissions updated.\n"
 }
 
+# start the server to generate configuration file - Prep for post install parameters [30 second start then kill]
+function first_start_config_ops()
+{
+	if[! -d "/root/Zomboid/"]
```

Comment:
We should be able to stick with the standard `steam` user for this.

Reply:
Need to see why the configuration is being placed under /root/Zomboid upon start

Reply:
[See comment](https://github.com/Renegade-Master/zomboid-dedicated-server/pull/2/files#r777268098)